### ISSUE TITLE
Inline Code Highlighting for Themes

### ISSUE THREAD

**Issue by aymanfarhat** [highlightjs/highlight.js#208]
Nice work on the themes but here is an issue I am facing. In my webpage I have both block and inline code(one line wrapped within text) which I would like to highlight. 

The convention for displaying inline code is via only a &lt;code&gt; tag while a block of code via &lt;pre&gt;&lt;code&gt; I have noticed that in the themes there is accounting only for block tags where all the selectors are preceded with a pre. Thus the style wont get applied on inline code tags also if they're nested into any container other than pre. 

So basically the current themes work only on the default init and not something like $('code').each(function(i,e){hljs.highlightBlock(e);}); 

Is there a certain reason for that? Why not just have code as the selector in the themes instead of pre code?

Thanks.

Edit: Here is an example of what I'm talking about https://gist.github.com/aymanfarhat/5355383 works well in my case. If you wish I would be ready to contribute to the project in converting the other themes.
